src/realm/obj.cpp

```diff
@@ -898,9 +898,9 @@ void out_mixed_xjson(std::ostream& out, const Mixed& val)
             out << "\"}";
             break;
         case type_UUID:
-            out << "{\"$uuid\": \"";
+            out << "{\"$binary\": \"";
             out << val.get<UUID>();
-            out << "\"}";
+            out << "\", \"subType\": \"04\"}}";
```

Comment:
does this not work? I thought @pavel-ship-it looked it up and the server does actually support the `$uuid` syntax (though I think it is not part of the bson standard) https://github.com/realm/realm-core/pull/4107

Reply:
This looks like it got in here from a merge, I'll revert